src/components/navigationBar/styles.ts

```diff
@@ -0,0 +1,102 @@
+import styled from "styled-components";
+import theme from "../../theme";
+import { ExpandedProps, Props } from "./types";
+
+export const ItemsContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  width: 100%;
+  height: 100%;
+`;
+
+export const ItemsSection = styled.div<{ stickToBottom?: boolean }>`
+  display: flex;
+  flex-wrap: no-wrap;
+  flex-direction: column;
+  row-gap: ${theme.spacing.spacing04};
+  width: 100%;
+  border-top: ${({ stickToBottom }) =>
+    stickToBottom ? `solid 1px ${theme.color.border.border01}` : "inherit"};
+  padding-top: ${({ stickToBottom }) =>
+    stickToBottom ? theme.spacing.spacing03 : "0"};
+  margin-top: ${({ stickToBottom }) =>
+    stickToBottom ? theme.spacing.spacing06 : "0"};
+`;
+
+export const NavigationBarWrapper = styled.nav<
+  Omit<Props, "items"> & ExpandedProps
+>`
+  display: flex;
+  flex-wrap: no-wrap;
+  justify-content: flex-start;
+  margin: 0;
+  flex-direction: column;
+  position: ${({ position }) => position};
+  top: 0;
+  bottom: 0;
+  left: 0;
+  width: ${({ expanded }) => (expanded ? "14rem" : theme.spacing.spacing07)};
+  background-color: ${({ backgroundColor }) => backgroundColor};
+  padding: ${theme.spacing.spacing05}
+    ${({ expanded }) =>
+      expanded ? theme.spacing.spacing04 : theme.spacing.spacing03}
+    ${theme.spacing.spacing04};
```

Comment:
Maybe it's better to be explicit here (padding-left/right/top/bottom), it could improve readability.

Reply:
👍